client-mr/src/main/java/org/apache/hadoop/mapred/RssMapOutputCollector.java

```diff
@@ -121,7 +130,8 @@ public void init(Context context) throws IOException, ClassNotFoundException {
         numMaps,
         isMemoryShuffleEnabled(storageType),
         sendThreadNum,
-        sendThreshold);
+        sendThreshold,
+        1024000L);
```

Comment:
it should be configured instead of 1024000L

Reply:
sorry, I'll fix this.